city_scrapers/spiders/ind_indygo.py

```diff
@@ -36,52 +47,45 @@ def parse(self, response):
 
             yield meeting
 
-    def _parse_title(self, item):
+    def _parse_title(self, date_item):
         """Parse or generate meeting title."""
         title = "IndyGo Board"
         return title
 
-    def _parse_description(self, item):
+    def _parse_description(self, date_item):
         """Parse or generate meeting description."""
-        string = item.css("::text").get()
-        string_split = string.split(" ")
-        if len(string_split) > 3:
-            description = string.split(" ", 3)[3]
-        else:
-            description = ""
+        description = ""
         return description
 
-    def _parse_classification(self, item):
+    def _parse_classification(self, date_item):
         """Parse or generate classification from allowed options."""
         return BOARD
 
-    def _parse_start(self, item):
+    def _parse_start(self, date_item, meeting_year, meeting_time):
         """Parse start datetime as a naive datetime object."""
-        string = item.css("::text").get()
-        start_date = string.split(" ")[1] + " " + string.split(" ")[2] + " 2023"
-        start_time = "17:00:00"
-        return parser().parse(start_date + " " + start_time)
+        meeting_date = (date_item.css("::text").get()).split("–")[0]
+        return parser().parse(meeting_date + " " + meeting_year + " " + meeting_time)
 
-    def _parse_end(self, item):
+    def _parse_end(self, date_item):
         """Parse end datetime as a naive datetime object. Added by pipeline if None"""
         return None
 
-    def _parse_time_notes(self, item):
+    def _parse_time_notes(self, date_item):
         """Parse any additional notes on the timing of the meeting"""
-        return "Board meetings are set for 5:00PM unless otherwise noted in meeting description. Please double check the website before the meeting date."  # noqa
+        return ""  # noqa
 
-    def _parse_all_day(self, item):
+    def _parse_all_day(self, date_item):
         """Parse or generate all-day status. Defaults to False."""
         return False
 
-    def _parse_location(self, item):
+    def _parse_location(self, date_item):
```

Comment:
In the vein of [this comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1452751539), we can make this code a bit cleaner.

Since the location dict is essentially a constant in this case we should delete the method and assign the value to a class variable. It makes the code easier to read because a contributor can see instantly that this value is a constant. Eg.
```
class IndIndygoSpider(CityScrapersSpider):
    name = "ind_indygo"
    agency = "Indianapolis Indygo Board"
    timezone = "America/Detroit"
    start_urls = ["https://www.indygo.net/about-indygo/board-of-directors/"]
    location = # add location value
```
and then assign the value to the Meeting object like so:
```
meeting = Meeting(
    ...
    location=self.location,
    ...
```
You can see an example of this pattern [here](https://github.com/City-Bureau/city-scrapers/blob/19fd52d76aa314734f620c4dafa9be0a0228ba81/city_scrapers/spiders/chi_il_medical_district.py).

At this point we don't have much of a convention across our codebase – we often do return values from our parse methods even if they're essentially just returning constants – but to my mind we should start using class variables more consistently for better code readability. You could take a similar approach to some other values, eg "IndyGo Board", but it's such a short string I think it's probably fine to just assign it to the Meeting object directly. The same for indistinctive values like None or "".

I'd be appreciative if you can review other areas of this file and the other files you've changed and use class vars where it seems appropriate. Don't forget to delete unused methods.